src/main/java/com/example/Ecommerce/entity/CartItemStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.example.Ecommerce.entity;
+
+public enum CartItemStatus {
```

Comment:
If no other statuses will be added, then it can be called `CartItemReservationStatus`